### PR TITLE
bugfix/20548-resize-after-fullscreen

### LINKED ISSUE #20548: Chart stops scaling after exiting fullscreen
#### Expected behaviour
After exiting fullscreen mode, the chart should adapt to the window size.

#### Actual behaviour
After exiting fullscreen mode, the chart stays the same, regardless of the window size.

#### Live demo with steps to reproduce
1. Open the full screen mode via context menu
2. Close the full screen mode via context menu
3. Try to resize the chart frame

[https://jsfiddle.net/BlackLabel/mLcvy71k/](https://jsfiddle.net/BlackLabel/mLcvy71k/)

#### Product version
Introduced in Highcharts v11.3.0

_Internal note:_ bisected to [db37d46](https://github.com/highcharts/highcharts/commit/db37d463502f974f9a48f8454262a86deb96117b)

#### Workaround

Set `chart.isResizing = 0` after the `close()` method is finished: [https://jsfiddle.net/BlackLabel/L2p59y34/](https://jsfiddle.net/BlackLabel/L2p59y34/)

### PR DESCRIPTION
**Pull request by Xillez**
Fixed #20548, chart not resizing after closing fullscreen.

---

Asana task: https://app.asana.com/0/1204848917817812/1206476497012682/f

### CODE REVIEW

**Review by hubertkozik — APPROVED** [review]

File: samples/unit-tests/fullscreen/toggle/demo.js
Comment (KacperMadej) [inline samples/unit-tests/fullscreen/toggle/demo.js:55]: This is a quite long waiting time. I understand it needs to be more than 100ms for the setSize in reflow to finish as it runs after a fixed (100ms) interval, but maybe this could be more stable if the below assertion were running after chart's `endResize` event that is fired when the setSize finishes? Having a fallback to some fixed timeout in case `reflow` doesn't call `setSize` at all would be good.

File: samples/unit-tests/fullscreen/toggle/demo.js
Comment (KacperMadej) [reply samples/unit-tests/fullscreen/toggle/demo.js:44]: If you could add a new issue about this with your comment that should be enough for the work here IMHO.

**Review by KacperMadej — APPROVED** [review]
One last comment from me left. Other than that, all seems fine, so approving in advance 👍

**Review by TorsteinHonsi — COMMENTED** [review]
Pending CI tests after merge

**Pull request merged**
